Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml

```diff
@@ -0,0 +1,346 @@
+        - Katana
+        - WeaponLaserCannon
+        - MuteToxinChemistryBottle
+        - ClothingOuterHardsuitCap
+        - ClothingShoesBootsMagBlinding
+        - TurboItemRechargerCircuitboard
+        - WeaponRevolverDeckard
+        - MedicatedSuture
+        - FoodPizzaArnold
+        - PinpointerUniversal
+        - Claymore
+      rareChance: 0.3
+
+- type: entity
+  id: RandomRarelegalLoot
+  name: Random Rare Legal Loot
+  suffix: Spawner 
+  parent: MarkerBase
+  components:
+    - type: Sprite
+      layers:
+        - state: red
+        - sprite: Objects/Misc/bedsheets.rsi
+          state: sheetUSA
+    - type: RandomSpawner
+      prototypes:
+        - ClothingBeltSheathFilled
+        - SpaceCash10000
+        - WeaponAntiqueLaser
+        - WeaponForceGun
+        - PenCentcom
+        - ClothingOuterArmorPodWars
+        - ClothingHeadHelmetPodWars
+        - ClothingOuterCoatSpaceAsshole
+        - UnholyHalberd
+        - ClockworkShield
+        - MirrorShield
+        - BedsheetUSA
+        - HypoMini
+        - HypoMiniLimitedEdition
+        - ClothingBackpackHolding
+        - ClothingShoesBootsMagAdv
+        - OreBagOfHolding
+        - ClothingShoesBootsSpeed
+        - ClothingBackpackDuffelHolding
+        - ClothingBackpackMessengerHolding
+        - MedkitCombatFilled
+        - TrashBagOfHolding
+        - AdvMopItem
+        - HandheldCrewMonitor
+        - WeaponAdvancedLaser
+        - VariedXenoArtifactItem
+        - ClothingMaskArcadia
+        - ClothingShoesBootsMagCombatFilled
+        - WeaponLaserCannon
+        - ClothingEyesHudMedSec
+        - PowerCellMicroreactor
+        - MuteToxinChemistryBottle
+        - SmokeGrenade
+        - ClothingShoesBootsMagBlinding
+        - TechnologyDiskRare
+        - Truncheon
+        - TurboItemRechargerCircuitboard
+        - ResearchDisk10000
+        - MedicatedSuture
+        - RegenerativeMesh
+        - FoodPizzaArnold
+        - PinpointerUniversal
+      chance: 1
+
+- type: entity
+  id: RandomEventIllegalLoot 
+  name: Random Illegal Loot
+  suffix: Spawner 
+  parent: MarkerBase
+  components:
+    - type: Sprite
+      layers:
+        - state: red
+        - sprite: Objects/Misc/bedsheets.rsi
+          state: sheetsyndie
+    - type: RandomSpawner
+      prototypes:
+      - DnaScramblerImplanter
+      - CyberPen
+      - CigPackSyndicate
+      - WeaponRevolverPythonAP
+      - ClothingEyesGlassesOutlawGlasses
+      - SyndicatePersonalAI
+      - EnergySword
+      - WeaponPistolCobra
+      - EmpGrenade
+      - Stimpack
+      - ClusterGrenade
+      - EncryptionKeyBinary
+      - EnergyShield
+      - SyndieMiniBomb
+      - PenExplodingBox
+      - GrenadeShrapnel
+      - GrenadeIncendiary
+      - ExGrenade
+      - C4
+      - SmokeGrenade
+      - GrenadeFlashBang
+      - ClothingShoesBootsMagSyndie
+      - ClothingBackpackDuffelSyndicateEVABundle
+      - ClothingUniformJumpskirtOperative
+      - ClothingOuterVestWeb
```

Comment:
Not sure how this qualifies as illegal, it has no syndicate markings and can't hold more than the merc variant.  It's really good because of the resistances on it, but this strikes me more as a rare legal item rather than a common illegal item.

Reply:
thats because I think in upstream the vest web is an syndi item, when I did the illegal spawner I gone through the uplink and a bunch of surplus crates

Reply:
made legal